AzureFunctionForSplunk/ActivityLogs/AzMonActivityLog.cs

```diff
@@ -0,0 +1,67 @@
+﻿//
```

Comment:
This file seems redundant with the addition of ActivityLogs/ActivityLog.cs.

Reply:
Your right, forgotten to take them out after changes I think. I made the same mistake for metrics and diagnostics.

Reply:
Thanks.